mypy_self_check.ini

```diff
@@ -20,3 +20,12 @@ always_false = MYPYC
 plugins = misc/proper_plugin.py
 python_version = 3.6
 exclude = mypy/typeshed/|mypyc/test-data/|mypyc/lib-rt/
+allow_any_expr = True
+allow_any_explicit = True
+allow_any_decorated = True
+allow_subclassing_any = True
+no_warn_return_any = True
+no_warn_no_ignore_code = True
+no_warn_unreachable = True
+implicit_reexport = True
+disallow_redefinition = True
```

Comment:
why enable these?

Reply:
For compatibility with upstream. We now do two runs, one with loose strictness, and one with strict + baseline.

Reply:
This isn't actually needed if we baseline from the head of upstream during the rebase, but that sounds painful to do manually. TODO would be to write a script to do it.